pkg/registry/rbac/rest/storage_rbac.go

```diff
@@ -334,3 +342,40 @@ func primeAggregatedClusterRoles(clusterRolesToAggregate map[string]string, clus
 
 	return nil
 }
+
+// primeSplitClusterRoleBindings ensures the existence of target ClusterRoleBindings
+// by copying Subjects, Annotations, and Labels from the specified source
+// ClusterRoleBinding, if present.
+func primeSplitClusterRoleBindings(clusterRoleBindingToSplit map[string]rbacapiv1.ClusterRoleBinding, clusterRoleBindingClient rbacv1client.ClusterRoleBindingsGetter) error {
+	for existingBindingName, clusterRoleBindingToCreate := range clusterRoleBindingToSplit {
+		// If source ClusterRoleBinding does not exist, do nothing.
+		existingRoleBinding, err := clusterRoleBindingClient.ClusterRoleBindings().Get(existingBindingName, metav1.GetOptions{})
+		if apierrors.IsNotFound(err) {
+			return nil
```

Comment:
the `return nil` lines inside the loop should be `continue` statements

Reply:
good catch